models/segment.schema.js

```diff
@@ -0,0 +1,20 @@
+const mongoose = require('mongoose');
+
+const segmentSchema = new mongoose.Schema({
+  _id: String,
+  name: {type: String, required: true, unique: true},
+  description: String,
+  geofence_area: {
+    latitude: Number,
+    longitude: Number,
+  },
+  deadline: Date,
+  volunteers: [
+    {
+      id: Schema.Types.ObjectId,
```

Comment:
🚫 **[eslint]** <[no-undef](https://eslint.org/docs/rules/no-undef)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'Schema' is not defined.